test_util/cluster_api.py

```diff
@@ -200,6 +200,7 @@ def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves,
                 configured if its value is "true".
             provider: onprem, azure, or aws
             auth_enabled: True or False
+            mesos_user: account that mesos is running under
```

Comment:
Really, it's the "default linux user that Marathon will launch its tasks as". Other frameworks can be configured to launch their tasks as different linux users, and even Marathon apps can override the default to launch tasks as other users.